test/color/test_ycbcr.py

```diff
@@ -0,0 +1,248 @@
+        data = torch.tensor([[[0.7925, 0.8704, 0.2771, 0.3279, 0.0193],
+                              [0.2483, 0.6212, 0.9859, 0.5044, 0.5621],
+                              [0.5762, 0.3959, 0.2931, 0.2669, 0.0243],
+                              [0.6989, 0.0529, 0.8344, 0.6523, 0.8980],
+                              [0.5181, 0.9341, 0.2172, 0.0520, 0.7266]],
+                             [[0.8413, 0.0284, 0.3625, 0.8864, 0.5595],
+                              [0.3791, 0.0235, 0.4251, 0.0619, 0.5270],
+                              [0.3516, 0.8005, 0.9571, 0.4113, 0.6119],
+                              [0.0632, 0.8836, 0.0261, 0.1550, 0.4923],
+                              [0.2332, 0.7044, 0.9514, 0.2443, 0.2818]],
+                             [[0.6899, 0.2063, 0.3179, 0.8989, 0.4378],
+                              [0.0384, 0.5230, 0.6416, 0.9749, 0.7863],
+                              [0.8577, 0.3115, 0.2375, 0.5446, 0.9837],
+                              [0.3213, 0.6618, 0.5977, 0.3999, 0.4962],
+                              [0.1385, 0.5831, 0.9756, 0.8714, 0.8017]]])  # 3x5x5
+
+        y_expected = torch.tensor([[0.8094629, 0.30041942, 0.33188093, 0.7208117, 0.38409105],
+                                   [0.30114722, 0.2591345, 0.6174494, 0.29830942, 0.5670594],
+                                   [0.47643244, 0.623786, 0.6765313, 0.38332558, 0.47857922],
+                                   [0.2826805, 0.60994685, 0.33298355, 0.3316514, 0.6140681],
+                                   [0.30756146, 0.75926465, 0.7346588, 0.25828302, 0.4740529]])
+        cr_expected = torch.tensor([[0.48791525, 0.9063825, 0.46094114, 0.21983841, 0.23989305],
+                                    [0.46231723, 0.7581379, 0.7626976, 0.6469568, 0.496467],
+                                    [0.5711212, 0.33752257, 0.22661471, 0.4169921, 0.17608929],
+                                    [0.7967522, 0.10283372, 0.85753804, 0.72865105, 0.70245713],
+                                    [0.65009415, 0.6246666, 0.13107029, 0.35291404, 0.6800583]])
+        cb_expected = torch.tensor([[0.43258172, 0.4469004, 0.492101, 0.60042214, 0.53028214],
+                                    [0.35180065, 0.64882994, 0.51362985, 0.8815981, 0.62366176],
+                                    [0.71505237, 0.32387614, 0.2523859, 0.5909421, 0.78490996],
+                                    [0.52178127, 0.529264, 0.64931786, 0.53848785, 0.43353173],
+                                    [0.4046721, 0.40065458, 0.6358658, 0.8458253, 0.6847739]])
+
+        # Kornia
+        f = kornia.color.RgbToYcbcr()
+        result = f(data)
+
+        y = result[0, :, :]
+        cb = result[1, :, :]
+        cr = result[2, :, :]
+
+        assert_allclose(y, y_expected)
+        assert_allclose(cb, cb_expected)
+        assert_allclose(cr, cr_expected)
+
+    def test_batch_rgb_to_ycbcr(self):
+        data = torch.tensor([[[0.7925, 0.8704, 0.2771, 0.3279, 0.0193],
```

Comment:
why not randomly generated with fixed/frozen seed?

Reply:
What are the benefits of such approach?

Reply:
that you do not need to insert it manually and it saves space...

Reply:
@Borda it also comes from the motivation of testing against results obtained with known frameworks such as opencv

Reply:
well, what does it mean obtained results?

Reply:
as far as I understood this hardcoded values come from using the matched function in opencv

Reply:
sounds like magic and guessing, then I think that it should be written as a comment in the code